evalml/tests/component_tests/test_lsa.py

```diff
@@ -47,46 +46,15 @@ def test_lsa_no_text():
     assert len(X_t.columns) == 2
 
 
-def test_some_missing_col_names(text_df, caplog):
```

Comment:
At first, I was confused as to why we were getting rid of the warning but I think it makes sense to do so since the user is no longer provided the text column names as input.

Reply:
Yup, we no longer need to check since we're not relying and checking the user's input :3